app/server/__init__.py

```diff
@@ -101,6 +101,19 @@ def enable_form_raw_cache():
 
 
 def register_blueprints(app):
+    @app.before_first_request
+    def before_first_request():
```

Comment:
It's not exactly a dependency IMO-- this code does make a celery call, but all that call does on the worker-end is create a new database row for the filter object if it doesn't already exist. Going the other way, the callback address is also customizable. You could fully eliminate the worker and it would just write to a queue and do nothing. Similarly, the sync code is fairly easy for new projects to interface with, since all that it requires is a new database row on the worker DB.  
   
Additionally, I think this way is the closest we can get to solving this particular problem while maintaining our design principles of the worker being dumb and having as generic as feasible blockchain logic, and the api not having to care about those details.